include/seqan3/alignment/matrix/all.hpp

```diff
@@ -18,7 +18,7 @@
  * \see alignment
  */
 
-#include <seqan3/alignment/matrix/alignment_coordinate.hpp>
+#include <seqan3/alignment/matrix/detail/advanceable_alignment_coordinate.hpp>
```

Comment:
the all-header shouldn't include detail stuff?

Reply:
Hmm I already removed that :)